src/hathor.c

```diff
@@ -0,0 +1,153 @@
+#include <stdint.h>   // uint*_t
+#include <stddef.h>   // size_t
+#include <stdbool.h>  // bool
+#include <string.h>   // memmove
+
+#include "os.h"
+
+#include "hathor.h"
+
+// #include "transaction/types.h"
+
+// Path prefix of 44'/280'
+const uint32_t htr_bip44[] = { 44 | 0x80000000, HATHOR_BIP44_CODE | 0x80000000 };
+
+void init_bip32_full_path(const uint32_t *in, uint8_t inlen, uint32_t *out) {
```

Comment:
Should we use the bip32 struct with length and path? It seems dangerous to write on `out` without checking its length. I'm not sure whether a buffer overflow here can be exploited.

Reply:
We do not use this method anymore